### PR TITLE
Add GHA CI workflow

### PR DESCRIPTION
**Pull request by internalsystemerror**
Converts CI workflow from Travis to GHA using: https://gist.github.com/weierophinney/b003e50c3c2667d08076caf31ebd36a4

### CODE REVIEW

File: .laminas-ci.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "extensions": [
+        "curl",
+        "dom",
+        "gd",
+        "libxml"
```
Comment (boesing) [inline .laminas-ci.json]: Afaik, `libxml` is bundled with PHP.
Same for `gd`, `dom` and `curl`.

Try to drop these to see if build passes. 👍🏼

File: README.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 # laminas-diactoros
 
-[![Build Status](https://travis-ci.com/laminas/laminas-diactoros.svg)](https://travis-ci.com/laminas/laminas-diactoros)
-[![Coverage Status](https://coveralls.io/repos/github/laminas/laminas-diactoros/badge.svg)](https://coveralls.io/github/laminas/laminas-diactoros)
+[![Build Status](https://github.com/laminas/laminas-diactoros/workflows/Continuous%20Integration/badge.svg)](https://github.com/laminas/laminas-diactoros/actions?query=workflow%3A"Continuous+Integration")
```
Comment (boesing) [inline README.md]: Could you apply the changes as described here? I think thats more future proof than using the workflow name.
https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-workflow-file-name

**Review by boesing — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (internalsystemerror) [reply README.md]: done

File: .laminas-ci.json (hunk 1 above)
Comment (internalsystemerror) [reply .laminas-ci.json]: You are correct in that curl, dom and libxml come bundled but GD is not. I have updated the configuration to reflect this.

**Pull request merged**